CHANGES.md

```diff
@@ -14,7 +14,24 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+  ### Changes
+
+  - Add request caching settings in `TWitcher` INI configuration to work with `Magpie` to help reduce permission and
+    access control computation time.
+    
+  - Add `magpie` logger under `Twitcher` INI configuration to provide relevant logging details provided 
+    by `MagpieAdapter` it employs for service and resource access resolution.
```

Comment:
rephrase "provide relevant logging details provided by `MagpieAdapter` it employs"